python/ray/worker.py

```diff
@@ -438,9 +434,11 @@ def get_gpu_ids():
     Returns:
         A list of GPU IDs.
     """
+
+    # TODO(ilr) Handle inserting resources in local mode
```

Comment:
Don't think this is very important either

Reply:
Should I remove the TODO?  There are a few github issues around it. I was planning to revisit this in a future PR, not this one.